### PR TITLE
feat(content): add authors and posts

### PR DESCRIPTION
**Pull request by Aharonla**

### CODE REVIEW

File: data/TODO.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+### Add posts
```
Comment (chiptus) [inline data/TODO.md:1]: remove file

File: data/blog/_categories/miscellaneous.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+title: 'Miscellaneous '
```
Comment (chiptus) [inline data/blog/_categories/miscellaneous.md:2]: remove whitespace

File: data/blog/how-to-run-a-node-on-the-tezos-blockchain-part-1.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  - tezos
+category: Devs
+published: true
+author: contact@tezos.co.il
+date: 2020-02-16T11:54:28.076Z
+description: How to Run a Node on the Tezos Blockchain (Part 1)
+featuredImage: /assets/how-to-run-a-node-part1_cover-300x90.png
+---
+
+#### <span style="text-decoration: underline;">**Note: This is _Part 1_ of  “How to Run a Node on the Tezos Blockchain” Series**</span>
```
Comment (chiptus) [inline data/blog/how-to-run-a-node-on-the-tezos-blockchain-part-1.md:14]: remove html. this goes for other posts too

File: data/blog/how-to-run-a-node-on-the-tezos-blockchain-part-1.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+To run a node building from sources, please read [part 2](https://tezos.co.il/701-2/)
+
+To run a node using a snapshot, please read[ part 3](https://tezos.co.il/719-2/)
```
Comment (chiptus) [inline data/blog/how-to-run-a-node-on-the-tezos-blockchain-part-1.md:18]: remove whitespace from link
```suggestion
To run a node using a snapshot, please read [part 3](https://tezos.co.il/719-2/)
```

File: data/blog/playing-with-smartpy-smart-contracts-part-1.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
+---
+templateKey: _blog-post
+title: Playing with SmartPy Smart Contracts (Part 1)
```
Comment (chiptus) [inline data/blog/playing-with-smartpy-smart-contracts-part-1.md:3]: we have a duplicate for this, remove the duplicate

File: data/blog/sample-post.md [hunk 6]
```diff
@@ -4,21 +4,17 @@ title: Playing with SmartPy Smart Contracts (Part 1)
 tags:
```
Comment (chiptus) [inline data/blog/sample-post.md:4]: remove this sample post

**Review by chiptus — CHANGES_REQUESTED** [review]
very good. remove some old files and we can merge this.

File: data/blog/how-to-run-a-node-on-the-tezos-blockchain-part-1.md (hunk 3 above)
Comment (Aharonla) [reply data/blog/how-to-run-a-node-on-the-tezos-blockchain-part-1.md:14]: I tried to find a different way to get an underline. this was the only way I found

File: data/blog/how-blockchain-can-help-banks-and-financing.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+In their effort to combat fraud, financial institutions have set up well funded fraud prevention teams to protect data and prevent money laundering. While these initiatives have introduced real time information sharing and built predictive models, it has also has lead to higher costs and longer onboarding times, which results in increased friction and lowered customer experience. As banks shift to digital services,  they now have more access to customer and transaction data than ever before, yet are struggling to prevent fraud & reduce costs while increasing customer experience. So what can banks do to streamline these processes?
+
+### **The Answer: Blockchain**
+
+![http://www.mnazureusergroup.com/wp-content/uploads/2018/07/f4bfd768-4990-4855-8abd-a7497678f415-300x225.jpg](https://tezos.co.il/wp-content/uploads/2020/01/anti-fraud.jpeg)
```
Comment (chiptus) [inline data/blog/how-blockchain-can-help-banks-and-financing.md:25]:
```suggestion
![Blockchain Anti-Fraud](https://tezos.co.il/wp-content/uploads/2020/01/anti-fraud.jpeg)  
```

**Pull request merged**